src/App.tsx

```diff
@@ -1,26 +1,30 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from 'react'
+import logo from './logo.svg'
+import './App.css'
+import axios from 'axios'
+
+const backendAddr = 'http://127.0.0.1:3000/api/v1/ping'
+
+function onClick() {
+    axios.get(backendAddr).then(res => {
+        console.log(res)
+        let resultDiv = document.getElementById("result")
+        if (resultDiv) {
+            resultDiv.innerText += res.data['message']
+        }
+    })
+}
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    return (
+        <div className="App">
+            <header className="App-header">
+                <img src={logo} className="App-logo" alt="logo"/>
+                <button onClick={onClick}>Ping</button>
+                <div id="result"/>
+            </header>
+        </div>
+    )
 }
```

Comment:
You should not keep views inside App.js rather routes, configs etc. but for the presentation purpose I can handle that.

Reply:
what

Reply:
how